mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB="$(mktemp --suffix -snapshot.db)"
+SNAPSHOT_SETTINGS_PATH="$(mktemp --suffix -snapshot_settings.py)"
```

Comment:
Sure, I'll get this working, sorry I had forgotten that the `/` slashes are problematic.